client/src/components/SharedComponents/Signup/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import './signup.css'
+
+class Signup extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    confirm_password: '',
+    displayBio: false
+  }
+
+  onChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  ShowDisplayBio = () => {
+    this.setState({ displayBio: true })
+  }
+
+  lessDisplayBio = () => {
+    this.setState({ displayBio: false })
+  }
+
+  validateForm = () => {
+    return (
+      this.state.name.length > 2 &&
```

Comment:
You can meet a Chinese tourist named `Li`. I don't think you need to validate the user name.

Reply:
Maybe more or equal to 2.